kahuna/public/js/main.js

```diff
@@ -151,16 +151,29 @@ kahuna.run(['$rootScope', 'mediaApi',
 
 // Intercept 401s and emit an event
 kahuna.config(['$httpProvider', function($httpProvider) {
-    $httpProvider.interceptors.push('httpUnauthorisedInterceptor');
+    $httpProvider.interceptors.push('httpErrorInterceptor');
 }]);
 
-kahuna.factory('httpUnauthorisedInterceptor',
+kahuna.factory('httpErrorInterceptor',
                ['$q', '$rootScope', 'httpErrors',
                 function($q, $rootScope, httpErrors) {
     return {
         responseError: function(response) {
-            if (response.status === httpErrors.unauthorised.errorCode) {
-                $rootScope.$emit('events:error:unauthorised');
+            switch (response.status) {
+                case 0:
```

Comment:
That's slightly worrying, could we investigate some more? Were there any security errors in the console?